go-controller/pkg/node/gateway_init.go

```diff
@@ -240,6 +248,20 @@ func (n *OvnNode) initGateway(subnets []*net.IPNet, nodeAnnotator kube.Annotator
 	return err
 }
 
+func gatewayInterfaceFromExt(intfName string) (string, error) {
```

Comment:
what does "FromExt" mean?

Reply:
I don't remember anymore :( probably a legacy from my initial understanding of what was the implementation.
And agree, I'll try to find a better name.